packages/pwa/app/components/footer/index.jsx

```diff
@@ -30,11 +30,17 @@ import {HideOnDesktop, HideOnMobile} from '../responsive'
 import {getUrlWithLocale} from '../../utils/url'
 import LocaleText from '../locale-text'
 import {getSupportedLocalesIds} from '../../utils/locale'
+import useSite from '../../hooks/use-site'
+import {getL10nConfig} from '../../utils/utils'
+import {useLocation} from 'react-router-dom'
 
 const Footer = ({...otherProps}) => {
     const styles = useMultiStyleConfig('Footer')
+    const {pathname, search} = useLocation()
+    const l10nConfig = getL10nConfig(`${pathname}${search}`)
```

Comment:
Another example of where we have the site from the `useSite` hook and can easily access the localization config on it without having use use `getL10nConfig`.